swagger.yaml

```diff
@@ -132,6 +132,33 @@ paths:
         500:
           $ref: '#/responses/InternalError'
 
+  /images/{image_id}/upload:
+    post:
+      tags:
+        - "image"
+      summary: "Upload an image"
+      description: ". This call sets the image state to 'uploaded'."
```

Comment:
We should remove the '. ' at the beginning of the description

Reply:
Actually, it's missing the sentence before that I had intended to add. Thanks for spotting.